hack/run-conformance.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+echo "Running sonobuoy conformance e2e test suite"
+export KUBECONFIG=$(./kubevirtci kubeconfig)
+export TENANT_CLUSTER_NAME=${TENANT_CLUSTER_NAME:-kvcluster}
+export TENANT_CLUSTER_NAMESPACE=${TENANT_CLUSTER_NAMESPACE:-kvcluster}
+
+sonobuoy_release="https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.56.8/sonobuoy_0.56.8_linux_amd64.tar.gz"
+sonobuoy_result_tarball="results.tar.gz"
+sonobuoy_plugin="e2e"
+sonobuoy_pod_runner="${TENANT_CLUSTER_NAME}-sonobuoy-runner-$(echo $RANDOM | sha1sum | head -c 4)"
+sonobuoy_conformance_dir="/tmp/sonobuoy_conformance"
```

Comment:
does this results dir need to be configurable for any reason in order to attach the contents as prow artifacts?

Reply:
That was my assumption, I haven't figured out how to attach the artifacts to the prow job yet